tracer/src/Datadog.Trace.Tools.Runner/Checks/ProcessBasicCheck.cs

```diff
@@ -168,34 +146,85 @@ public static bool Run(ProcessInfo process, IRegistryService? registryService =
                 ok = false;
             }
 
-            ok &= CheckProfilerPath(process, runtime == ProcessInfo.Runtime.NetCore ? "CORECLR_PROFILER_PATH" : "COR_PROFILER_PATH", requiredOnLinux: true);
-            ok &= CheckProfilerPath(process, runtime == ProcessInfo.Runtime.NetCore ? "CORECLR_PROFILER_PATH_32" : "COR_PROFILER_PATH_32", requiredOnLinux: false);
-            ok &= CheckProfilerPath(process, runtime == ProcessInfo.Runtime.NetCore ? "CORECLR_PROFILER_PATH_64" : "COR_PROFILER_PATH_64", requiredOnLinux: false);
+            string corProfilerPathKey = runtime == ProcessInfo.Runtime.NetCore ? "CORECLR_PROFILER_PATH" : "COR_PROFILER_PATH";
 
-            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            process.EnvironmentVariables.TryGetValue(corProfilerPathKey, out var corProfilerPathValue);
+
+            bool isTracingUsingBundle = TracingWithBundle(corProfilerPathValue);
+
+            if (isTracingUsingBundle)
```

Comment:
My _only_ concern about this is it's still just a guess 🤔 It's _possible_ the customer installed to a location that fits this pattern, and then we would skip the IIS checks. I don't know how legitimate a worry that is tbh 🤔

Reply:
Can you pass in the directory of the executable to the `TracingWithBundle` method? The bundle is designed so that the `datadog` folder is copied directly into the application output folder. This means we should be able to test against the full path, not just an `EndsWith` check

Reply:
Yeah I wasn't sure how to do more than basically a guess if the customer set at least that configuration correctly, but at least  would stop the IIS checks if they did I thought and be less misleading.

Will look into the full path approach since sounds like a good idea.

Reply:
Added it, more details here: [https://github.com/DataDog/dd-trace-dotnet/pull/4605/commits/db2a3170c93eeb95b7509e688d42576e15af126d#r1324521240](https://www.google.com/url?q=https://github.com/DataDog/dd-trace-dotnet/pull/4605/commits/db2a3170c93eeb95b7509e688d42576e15af126d%23r1324521240&sa=D&source=docs&ust=1694615695373759&usg=AOvVaw3saY73tYYIco7R6GA4bx6t)